scripts/generateWebApiIndex.js

```diff
@@ -0,0 +1,58 @@
+const { XMLParser } = require("fast-xml-parser");
+const { get } = require("axios");
+const { writeFileSync } = require("fs");
+const { resolve } = require("path");
+
+const webAPIUrl = "https://developer.mozilla.org/en-US/docs/Web/API";
```

Comment:
It currently only indexes pages under this url. I am unaware of any APIs published elsewhere in MDN, though I also did not go looking for any.